app/src/main/java/com/hannesdorfmann/adapterdelegates3/sample/model/DiffCat.java

```diff
@@ -0,0 +1,55 @@
+package com.hannesdorfmann.adapterdelegates3.sample.model;
+
+import com.hannesdorfmann.adapterdelegates3.DiffItem;
+
+/**
+ * Created by six_hundreds on 15.06.18.
+ */
```

Comment:
Please remove this comment or make it more meaningful (I think removing it is better).